_test/conftest.sh

```diff
@@ -183,4 +183,15 @@ setup_file() {
 
   print_info "${status}" "${output}" "${cmd}" "${tmp}"
   [ "$status" -eq 0 ]
-}
\ No newline at end of file
+}
+
+@test "charts/kopf" {
+  tmp=$(helm_template "charts/tekton-demo")
```

Comment:
path is wrong, would expect the test name to be same as tmp, presume copy+paste error

Reply:
Yepp. Copy Pasta. Fixed.